webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -1,23 +1,35 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
-  </ds-button>
+  <div
+    v-if="totalNotifications <= 0"
+    class="notifications-menu-pointer"
+    @click.prevent="updateNotifications"
+  >
+    <ds-button class="notifications-menu" disabled icon="bell">{{ unreadNotifications }}</ds-button>
+  </div>
   <dropdown v-else class="notifications-menu" :placement="placement">
     <template slot="default" slot-scope="{ toggleMenu }">
-      <ds-button primary icon="bell" @click.prevent="toggleMenu">
-        {{ totalNotifications }}
+      <ds-button
+        primary
+        icon="bell"
+        @click.prevent="
+          toggleMenu()
+          updateNotifications()
```

Comment:
Actually I prefer not to `updateNotifications` when i click on the notifications. It could happen that the list gets updated just when I want to click on a notification.

Can't we just let the notifications update on their own?

Reply:
@roschaefer it might not be clear, but clicking on it to update only works if there are no `totalNotifications`, so it's not possible to update if you want to click on a notification

Reply:
If I need to have a quick conversation with somebody this way I am able to get the notifications menu as quick updated as possible. Also in case the automatic update stops for any reason.

I discussed this with @datenbrei and we agreed to do it this way.

To have the menu not updated while it is open we could say:
```
 @click.prevent="
          await updateNotifications()
          toggleMenu()
 "
```
But then there will be a delay before the menu opens …
I my eyes there is no big possible harm if the open menu sometimes updates after I click on it.

PS: The `toggleMenu` function is undefined in any method as far I have tried …

Reply:
I think my head explodes right now.

Up to you. If you feel it's a great idea to update the list when you click on notifications - feel free to do it that way. I would prefer to implement the smallest solution first and get some user feedback before moving on the more sophisticated solution. But up to you. I have confidence in your decisions :wink: